src/lib.rs

```diff
@@ -98,6 +98,7 @@ pub use message::{message_prettify, DEFAULT_COMMENT_CHAR};
 pub use note::{Note, Notes};
 pub use object::Object;
 pub use oid::Oid;
+pub use packbuilder::PackBuilder;
```

Comment:
This may also want to reexport `Stage` as I believe that's exported as part of the public API